core/state_processor.go

```diff
@@ -70,6 +70,9 @@ func (p *StateProcessor) Process(block *types.Block, statedb *state.StateDB, cfg
 	if p.config.DAOForkSupport && p.config.DAOForkBlock != nil && p.config.DAOForkBlock.Cmp(block.Number()) == 0 {
 		misc.ApplyDAOHardFork(statedb)
 	}
+	if p.config.IsOptimismMeadow(block.Time()) {
```

Comment:
Like the above DAO checks, we should probably change the pattern to effectively be an `==` check, rather than `>=`, to avoid adding a the code-size check on every single future L2 block build/verification.
The code does not execute during genesis, but we can add a bytecode check to the genesis generation tooling to ensure the code is contained at the address as expected.

Reply:
Would this require switching to a block number to schedule the fork, or are we comfortable during timestamp equality? or a range check?